### PR TITLE
web/satellite: access grant routing

### PR DESCRIPTION
**Pull request by mbouzi**
What: Access Grant routing

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue [hunk 1]
```diff
@@ -22,6 +22,9 @@
             >
                 <p class="project-dropdown__wrap__choice__unselected">{{ project.name }}</p>
             </div>
+						<div @click="toggleSelection" @click.prevent.stop="closeDropdown" class="project-dropdown__wrap__create-project__wrapper">
```
Comment (VitaliiShpital) [inline web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue:25]: not sure if this change should be here at all

File: web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue [hunk 2]
```diff
@@ -90,6 +93,13 @@ export default class ProjectDropdown extends Vue {
         return this.$store.getters.selectedProject;
     }
 
+        /**
+		 * Opens new project creation popup.
+		 */
+        public toggleSelection(): void {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue:99]: should be in another PR

File: web/satellite/src/components/accessGrants/steps/CreateAccessUplinkStep.vue [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="api-keys-area">
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/steps/CreateAccessUplinkStep.vue:5]: that's not an api-key-area

File: web/satellite/src/components/accessGrants/steps/CreateAccessUplinkStep.vue [hunk 4]
```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```
Comment (VitaliiShpital) [inline web/satellite/src/components/accessGrants/steps/CreateAccessUplinkStep.vue:1]: since this is a new file we should use 2020 here

File: web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue [hunk 5]
```diff
@@ -183,6 +193,20 @@ export default class ProjectDropdown extends Vue {
                     }
                 }
             }
+
+            &__create-project {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/header/projectsDropdown/ProjectDropdown.vue:197]: same about PR scope

File: web/satellite/src/router/index.ts [hunk 6]
```diff
@@ -203,18 +235,18 @@ router.afterEach(({name}, from) => {
...
 
     const selectedProjectName = store.state.projectsModule.selectedProject.name ?
-        `${store.state.projectsModule.selectedProject.name} | ` : '';
+    `${store.state.projectsModule.selectedProject.name} | ` : '';
 
     document.title = `${selectedProjectName + router.currentRoute.name} | ${store.state.appStateModule.satelliteName}`;
 });
 
 /**
- * if our route is a tab and has no sub tab route - we will navigate to default subtab.
```
Comment (VitaliiShpital) [inline web/satellite/src/router/index.ts:212]: unnecessary change?

File: web/satellite/src/router/index.ts [hunk 7]
```diff
@@ -27,8 +31,8 @@ const RegisterArea = () => import('@/views/register/RegisterArea.vue');
 Vue.use(Router);
 
 /**
- * RouteConfig contains information about all routes and subroutes
```
Comment (VitaliiShpital) [inline web/satellite/src/router/index.ts:30]: unnecessary change?

File: web/satellite/src/router/index.ts [hunk 8]
```diff
@@ -165,7 +176,28 @@ export const router = new Router({
                     name: RouteConfig.EditProjectDetails.name,
                     component: EditProjectDetails,
                 },
+                {
+                    path: RouteConfig.NameStep.path,
```
Comment (VitaliiShpital) [inline web/satellite/src/router/index.ts:180]: previous version was correct here. AccessGrant is a parent route of these 4 routes. So routes will be like this 
`/access-grants/access-create-name`. That means that we need AccessGrant component but not in the views folder like I mentioned before because AccessGrant is a child of dashboard but a parent of these 4 routes. Just return previous version of this file but change import path of AceessGrant.vue file

it should be imported like this `import AccessGrant from '@/components/accessGrants/AccessGrant.vue';`

**Review by VitaliiShpital — APPROVED** [review]

**Review by VitaliiShpital — APPROVED** [review]

**Review by VitaliiShpital — DISMISSED** [review]

**Review by NikolaiYurchenko — APPROVED** [review]

**Review by VitaliiShpital — APPROVED** [review]

**Review by NikolaiYurchenko — APPROVED** [review]
